src/assets/i18n/pt.json

```diff
@@ -3912,15 +3914,14 @@
   "iSCSI listen port": "",
   "iSCSI supports multiple  authentication methods that are used by the target to discover valid  devices. <i>None</i> allows anonymous discovery while <i>CHAP</i> and  <i>Mutual CHAP</i> require authentication.": "",
   "iXsystems does not audit or otherwise validate the contents of third-party applications catalogs.     It is incumbent on the user to verify that the new catalog is from a trusted source and that the third-party     properly audits its chart contents. Failure to exercise due diligence may expose the user and their data to     some or all of the following:<br/>    <ul>\n      <li>Malicious software</li>\n      <li>Broken services on TrueNAS host</li>\n      <li>Service disruption on TrueNAS host</li>\n      <li>Broken filesystem permissions on Host or within application</li>\n      <li>Unexpected deletion of user data</li>\n      <li>Unsafe service configuration in application</li>\n      <li>Degradation of TrueNAS host performance and stability</li>\n    </ul>": "",
-  "last run {date}": "",
+  "last run {date}": "Última execução {data}":",
```

Comment:
There is a minor typo here that prevents us from merging this PR.
Please adjust `"Última execução {data}":",` to `"Última execução {data}",`

Reply:
Nevermind, I was able to update it myself.